README.md

```diff
@@ -1 +1,86 @@
-# Restaurant-Page
\ No newline at end of file
+<h3 align="center">Restaurant Page - Javascript</h3>
+
+<div align="center">
+
+[![Status](https://img.shields.io/badge/status-active-success.svg)](https://github.com/umairarshadbutt/Restaurant-Page)
+[![Github Issues](https://img.shields.io/badge/GitHub-Issues-orange)](https://github.com/umairarshadbutt/Restaurant-Page/issues)
+[![GitHub2 Pull Requests](https://img.shields.io/badge/GitHub-Pull%20Requests-blue)](https://github.com/umairarshadbutt/Restaurant-Page/pulls)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](/LICENSE)
+</div>
+<p align="center">Restaurant Page in JavaScript(and Webpack) is a practice project, which includes managing the dynamically DOM manipulation.I used Javascript, HTML5, CSS3, Bootsrap and Webpack to implement it</p>
+
+
+
+
+## Live Link Demo
+
+[Live Link Demo](https://rawcdn.githack.com/umairarshadbutt/Restaurant-Page/feature/restaurant-page/dist/index.html#)
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootsrap
+- JavaScript
+- Webpack
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Text Editor (VSCode is suggested.)
+
+
+### Setup
+
+- Install [git](https://git-scm.com/downloads)
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/umairarshadbutt/Restaurant-Page`
```

Comment:
- [ ] Let's add more instructions on how to build the projects, and start the server.